steps/swift-package-manager-xcodeproj-for-mac/0.0.1/step.yml

```diff
@@ -0,0 +1,53 @@
+title: Swift Package Manager Xcodeproj for Mac
+summary: |
+  Runs Generate Xcode project on your macOS project with Swift Package Manager
+description: |
+  Runs Swift Package Manager's `swift package generate-xcodeproj` action for macOS app projects.
+website: https://github.com/kitasuke/bitrise-step-swift-package-manager-xcodeproj-for-mac
+source_code_url: https://github.com/kitasuke/bitrise-step-swift-package-manager-xcodeproj-for-mac
+support_url: https://github.com/kitasuke/bitrise-step-swift-package-manager-xcodeproj-for-mac/issues
+published_at: 2019-03-15T00:09:31.6352+09:00
+source:
+  git: https://github.com/kitasuke/bitrise-step-swift-package-manager-xcodeproj-for-mac.git
+  commit: 0361521c9b1feb7a392182c4c402592702001c86
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- macos
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/kitasuke/bitrise-step-swift-package-manager-xcodeproj-for-mac
+deps:
+  brew:
+  - name: go
+  check_only:
+  - name: xcode
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      Path where the Xcode project should be generated
+    is_expand: true
+    is_required: false
```

Comment:
Same kind question: can the step work without specifying output dir?

Reply:
Addressed